app/src/main/res/drawable/username_background.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    >
+
+    <solid
+        android:color="#fff"/>
+    <stroke android:width="1dp" android:color="#CED0D2" />
+    <corners android:radius="30dp"/>
+</shape>
```

Comment:
i think this is for the username box in toolbar right? this is is not needed anymore, since the layout is already achieved in the common too_bar_main